examples/pytorch/torchscript/MNIST/mnist_torchscript.py

```diff
@@ -0,0 +1,194 @@
+# pylint: disable=W0223
+
+from __future__ import print_function
```

Comment:
```suggestion
from __future__ import print_function
```

because we no longer support python 2